flexget/plugins/input/next_series_seasons.py

```diff
@@ -71,6 +71,8 @@ def on_task_input(self, task, config):
         if isinstance(config, bool):
             config = {}
 
+        DIFF_NO_SERIES_BEGIN = 15
```

Comment:
These should ideally go at the beginning of the module, outside if the plugin class. The idea is that constant values like these can be easily viewed when editing the file and kit hidden internally. 
Also I'm not too wild about their names but can't think of a better idea...